man/systemd.exec.xml

```diff
@@ -1184,7 +1184,11 @@
         details. Defaults to <option>shared</option>. Use <option>shared</option> to ensure that mounts and unmounts
         are propagated from the host to the container and vice versa. Use <option>slave</option> to run processes so
         that none of their mounts and unmounts will propagate to the host. Use <option>private</option> to also ensure
-        that no mounts and unmounts from the host will propagate into the unit processes' namespace. Note that
+        that no mounts and unmounts from the host will propagate into the unit processes' namespace.
+        If this is set to <option>slave</option> or <option>private</option>, any mounts created by spawned processes
+        will be unmounted after the completion of the current command line of <varname>ExecStartPre=</varname>,
```

Comment:
This is per command. "command line" is the language used in the documentation for ExecStart= and it's related commands, and there is a section named that, so I'm using the same phrase to mean the same thing.